### PR TITLE
876175-Added the print and Page Layout mode feature support to the custom toolbar sample in the PDF Viewer Github sample.

### PR DESCRIPTION
**Pull request by deepika134**
Task ID : https://dev.azure.com/EssentialStudio/Mobile%20and%20Desktop/_workitems/edit/876340/
Task - GitHub Demo for Page Layout Mode 

**Description :**
1.Added the print feature support to the custom toolbar sample in the PDF Viewer Github sample.
2. Added  the Pagelayoutmode support to the custom toolbar sample in the PDF Viewer Github sample

### CODE REVIEW

**Review by Deepak1799 — APPROVED** [review]
Approved the changes

File: Custom Toolbar/View/MainPage.xaml [hunk 1]
```diff
@@ -114,6 +115,14 @@
                         Command="{Binding ZoomInCommand}" IsEnabled="{Binding CanZoomIn}"
                         PropertyChanged="Button_PropertyChanged"/>
                 </HorizontalStackLayout>
+                <Grid Grid.Column="3" Margin="0,0,12,0" HorizontalOptions="{OnPlatform MacCatalyst=End, iOS=End, Android=End, WinUI=End}">
```
Comment (Deepak1799) [inline Custom Toolbar/View/MainPage.xaml:118]: Why onplatform is required if in all the platforms is is at the end?

**Review by Deepak1799 — CHANGES_REQUESTED** [review]
Requested changes

File: Custom Toolbar/View/MainPage.xaml (hunk 1 above)
Comment (manikandan18102001) [reply Custom Toolbar/View/MainPage.xaml:118]: I'll change sir

**Review by Deepak1799 — APPROVED** [review]
Approved

**Pull request merged**